relstorage/cache/storage_cache.py

```diff
@@ -195,7 +195,10 @@ def release(self):
 
         This is usually memcache connections if they're in use.
         """
-        clients = self.clients_local_first if self.clients_local_first is not _UsedAfterRelease else ()
+        clients = ()
+        if self.clients_local_first is not _UsedAfterRelease:
+            clients = self.clients_local_first
+
```

Comment:
Feel free to ignore, but I would have just wrapped the original expression in ()s and the line in some reasonable place.

But I think an if/else statement would be better than what you did, as it makes clearer that () and self.clients_local_first are alternatives.